custom-trait-examples/custom-trait-handwritten/README.md

```diff
@@ -0,0 +1,38 @@
+# Custom, handwritten trait
+Use this package to create a customizable [trait](https://smithy.io/2.0/spec/model.html#traits) package. This
+package will include both generated Java trait definitions and a handwritten Java trait definition.
+
+## Building
+To build this package run the following from the root of the package:
+
+```console
+./gradlew clean build
+```
+
+This will generate a `build` directory containing the build artifacts generated by
+gradle. The generated JAR file that should be used by downstream consumers can be
+found at `build/libs/custom-trait-handwritten.jar`
+
+
+## Customization
+To customize this package, start by updating the `settings.gradle.kts` file and update
```

Comment:
We are already talking about "custom traits". I find the use of "customization" confusing here.